tests/conftest.py

```diff
@@ -21,5 +21,29 @@ async def testdata_path() -> Path:
     return TEST_DIR / "data/"
 
 
+@pytest.fixture(scope="session")
+def tmp777_session_path(tmp_path_factory):
+    """
+    Makes a subdir of the tmp_path with 777 access rights
+    """
+
+    def _tmp_path_factory(relative_path: str):
+        tmp = tmp_path_factory.mktemp(relative_path)
+        tmp.chmod(0o777)
+        return tmp
+
+    yield _tmp_path_factory
+
+
+@pytest.fixture
+def tmp777_path(tmp_path):
+    tmp_path.chmod(0o777)
+    for parent in tmp_path.parents:
+        if "pytest" in parent.as_posix():
```

Comment:
I think this breaks if we use `pytest --basetemp $HOME/tmp`.
But it would work with `pytest --basetemp $HOME/tmp-pytest`, so I think there is no problem.

(nothing to do here)

Reply:
Will now loudly fail if basetemp does not contain `pytest` in its name.